src/components/table/LightTable.tsx

```diff
@@ -209,12 +209,20 @@ function LightTable({
     });
   };
 
-  const handleNullValue = () => <span className={classes.nullValue}>(empty)</span>;
+  const handleNullValue = () => (
+    <span key="emptyRow" className={classes.nullValue}>
+      (empty)
+    </span>
+  );
 
   const handleRepeatedValues = (values: Array<string>, columnName: string) => {
-    const cleanValues = values
-      .map((v) => (_.isNil(v) ? handleNullValue() : `${v}`))
-      .map((v, i) => <span key={`val-${i}`}>{v}</span>);
+    /* eslint-disable indent */
+    const cleanValues = _.isEmpty(values)
```

Comment:
Hmm...so if I read this right, an empty array would show up as `null`?  Do we want that behavior?

Reply:
Oh, sorry, so what you are showing is the case where the array is `[null]`, as opposed to `[]`? (Just cause it says 1 item)